### DISCUSSION TITLE
官方插件图库管理这个封面的功能好像有点不合理

### DISCUSSION THREAD

**Discussion by IshuaWang** [halo-dev/halo#7116] [Q&A] [answered]
如果设置封面，那么展示会显示封面而不是原图，但是这个时候好像没法看到原图是什么了，这就有点奇怪。感觉要不把这个改成缩略图，要不点击这个可以查看到原图是什么样子。

**Comment by guqing** [comment, marked as answer]
这是因为之前没有缩略图功能，因此添加了这个字段让用户手动填写小尺寸图片作为封面图，现在已经有了缩略图机制确实已经不需要这个字段了

使用此 issue 来代替此 discussion：https://github.com/halo-sigs/plugin-photos/issues/59 因此我将关闭它
